app/services/person_with_remote_fields.rb

```diff
@@ -0,0 +1,49 @@
+class PersonWithRemoteFields < SimpleDelegator
+  REMOTE_FIELDS = [
+    :email, :phone, :first_name, :last_name, :is_volunteer,
+    :employer, :occupation, :skills, :tags
+  ]
+  LOCAL_FIELDS = [
+    :email, :phone, :first_name, :last_name,
+    :address, :city, :zip, :is_volunteer
+  ]
+
+  def self.find_or_build(attributes)
+    email = attributes.symbolize_keys.delete(:email)
+    person = Person.find_or_initialize_by(email: email)
```

Comment:
I'm sure we'll need to find by more than just email, but want to when and where we find by other identifiers before implementing a more flexible method.